api/github.ts

```diff
@@ -132,7 +132,7 @@ async function status ({ owner, repo, ref = 'master', context }: PathArgs) {
   const resp = await restGithub(`repos/${owner}/${repo}/commits/${ref}/status`)
 
   let state = typeof context === 'string'
-    ? resp!.statuses.filter(st => st.context.startsWith(context))
+    ? resp!.statuses.filter(st => st.context.includes(context.toLowerCase()))
```

Comment:
Is there guarantee that context will always be lowercase?

Reply:
Sorry, what? The thing is... Ooooh, we should also `st.toLowerCase()`, got it.

Probably tomorrow will update it.

Reply:
If not, then I guess we better leave it as is 😉

Autocorrect might not always works as expected